### ISSUE TITLE
[css-text-decor-3] Hard to parse sentence

### ISSUE THREAD

**Issue by faceless2** [w3c/csswg-drafts#5158]
Fifth paragraph of https://drafts.csswg.org/css-text-decor-4/#line-decoration, just above the example, includes this text:

> The visibility property, text-shadow, filters, and other graphical transformations likewise affect text decorations as part of the text they’re drawn on, even if the decorations were specified on an ancestor box

I think that is trying to say that `visibility`, `text-shadow`, `filters` and other graphical transformations on an inline element are applied to any text-decoration drawn on that inline element, even if the text-decoration was specified on an ancestor box.

I know I'm not the only one who struggled with this sentence, as we have 12 tests trying to verify the exact opposite of what this sentence is trying to say. See https://github.com/web-platform-tests/wpt/pull/23949

**Comment by fantasai** [comment]
@faceless2 Fixed in https://github.com/w3c/csswg-drafts/commit/6857f42716297bd5aa51b7653e7af190fb509253 Let me know if the new wording works for you!

**Comment by faceless2** [comment]
It's everything I could hope for and more ;-)
